components/style/properties/longhand/font.mako.rs

```diff
@@ -411,7 +411,11 @@ ${helpers.single_keyword("font-variant-position",
 
         impl ToCss for FeatureTagValue {
             fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-                let s = format!("\"{}\" {}", self.tag, self.value);
+                let s = match self.value {
+                    1 => format!("\"{}\"", self.tag),
+                    0 => format!("\"{}\" off", self.tag),
+                    x => format!("\"{}\" {}", self.tag, x)
+                };
```

Comment:
Oh, also, given you're doing this, instead of allocating the new string and later calling `write_str`, I wonder if you could do `write!(dest, ...)` instead. It's a nit so if you don't have time to address it we can always file a follow-up :)

Thanks!